src/Components/Attractions.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from "react";
+
+const Attractions = (results) => {
+	// console.log(results.results.city.name);
+	const locationData = results.results;
+	const city = locationData.city.name;
+	const lng = locationData.city.coord.lon;
+	const lat = locationData.city.coord.lat;
+
+	useEffect(() => {
+		fetch(`https://api.opentripmap.com/0.1/en/places/radius?radius=1000&lon=${lng}&lat=${lat}9&rate=2&format=json&limit=15&apikey=5ae2e3f221c38a28845f05b6c06a2c73eb6f2499d3b1881719d6dd39`)
+			.then(res => res.json())
+			.then(data => {
+				console.log(data);
+			})
```

Comment:
Are we using this API's response?

Reply:
Yes, this is for our next assigned issue showing attractions....I pushed the changes to the main branch by mistake.

Reply:
Cool, revert these changes

Reply:
Done!